server/views/pages/overview/partials/additionalNeedsTab/_supportStrategiesSummaryCard.njk

```diff
@@ -20,9 +20,7 @@
             <div class="govuk-summary-list__row" data-qa="support-strategy-summary-list-row">
               <h3 class="govuk-heading-s govuk-!-margin-top-3"> {{ type | formatSupportStrategyTypeScreenValue | default(type, true) }} </h3>
               {% for supportStrategy in supportStrategies %}
-                <p class="govuk-body govuk-!-margin-top-3 app-u-multiline-text" data-qa="support-strategy-details">
-                  {{ supportStrategy.supportStrategyDetails }}
-                </p>
+                <p class="govuk-body govuk-!-margin-top-3 app-u-multiline-text" data-qa="support-strategy-details">{{ supportStrategy.supportStrategyDetails }}</p>
```

Comment:
Because of the css class used `app-u-multiline-text`, the content needs to be rendered on the same line as the opening `<p>` tag, otherwise it is rendered as a blank line which appears as being too much spacing! 🙄 

IE.
Previously it rendered like this:
```
<p class="govuk-body govuk-!-margin-top-3 app-u-multiline-text" data-qa="support-strategy-details">
   The content goes here
</p>
```

but it needs to render like this:
```
<p class="govuk-body govuk-!-margin-top-3 app-u-multiline-text" data-qa="support-strategy-details">The content goes here</p>

```